content/en/device-support/_index.md

```diff
@@ -10,32 +10,38 @@ weight: 100
 
 ## Overview
 
-Support for passkeys is currently rolling out across major operating systems and browsers, and will continue into 2023. This page will be updated as the ecosystem evolves. The [matrix below](#matrix) maps out the various features that support the passkey experience. Additional information about each platform is available in the [Reference section of Docs](/docs/reference/android).
+Support for passkeys is currently rolling out across major operating systems and browsers. This page will be updated as the ecosystem evolves. The [matrix below](#matrix) maps out the various features that support the passkey experience. Additional information about each platform is available in the [Reference section of Docs](/docs/reference/android).
 
 Passkeys created in **iOS or iPadOS** can be used on:
 
 - The same iPhone or iPad
-- iPhones and iPads using the same Apple ID and iCloud Keychain (synced automatically)
-- MacBooks using the same Apple ID and iCloud Keychain (synced automatically)
-- MacBooks using [FIDO Cross-Device Authentication](/docs/reference/terms/#cross-device-authentication-cda)
+- iPhones and iPads using the same Apple ID (synced automatically)
```

Comment:
Would shorten this to 'iOS devices' maybe?

Reply:
technically an iPad is not an iOS device, it's an iPadOS device.